packages/website/src/pages/ContentPage/PreviewPage.tsx

```diff
@@ -55,6 +58,8 @@ function PreviewPage({ file, edit = false, renderStackOffset = 0 }: IPreviewPage
     edit === true ? '/edit' : '/preview'
   );
 
+  const headSubtract = edit === true ? (!sidebarOpen ? 70 : 100) : 120;
```

Comment:
How about `const headSubtract = edit ? (!sidebarOpen ? 70 : 100) : 120;`.

Reply:
good point, that works when it is undefined.

I generally like to be more explicit about undefined/null. For example, if the comparison were against `false` it could be confused with undefined. That is why the linter, which is very picky, did not complain about this.

Reply:
Thanks, this code is changed now.

Reply:
Yeah, implicit conversions are sometimes really dangerous and confusing in JS. Type and default value saved us.